module/kvmfr.c

```diff
@@ -655,6 +655,8 @@ MODULE_LICENSE("GPL v2");
 MODULE_AUTHOR("Geoffrey McRae <geoff@hostfission.com>");
 MODULE_AUTHOR("Guanzhong Chen <quantum2048@gmail.com>");
 MODULE_VERSION("0.0.9");
```

Comment:
You should increase The module version to 0.0.12, and make a similar change in dkms.conf. This allows dkms to sense that an updated module is needed.